### ISSUE TITLE
pruning object detection models

### ISSUE THREAD

**Issue by abhishekbalu** [tensorflow/tensorflow#44341]
hi is there a way to prune object detection models that are frozen or in saved model format? I have taken a pre-trained model from the TensorFlow zoo and retrained with my own dataset. I would like to prune the model. I have tried several ways but nothing works. Can you help me out with the procedure to do so?. Thank you.

**Comment by ravikyram** [comment]
@abhishekbalu 

Please, refer the [link](https://towardsdatascience.com/pruning-convolutional-neural-networks-cae7986cbba8) and see if it helps you. Thanks!

**Comment by abhishekbalu** [comment]
I have tried to contact stack overflow forum but no luck. More over with recent works i have come across people facing the same issue. I have noticed there are many conversation in the forum which tells this is not possible and they link you have shared uses pytorch has the base model. But my question was not how to do it in tensorflow rather why tensorflow does not have this prune feature with for saved model.pb files. When i tried to explore these models i was not able to explore the model using summary(). I would like to work on this and figure out a way to prune the model.pb files on tensorflow directly. The tensorflow model zoo gives us the opportunity to work with object detection but running these models on a embedded platform but a overhead on resource. So  pruned models can make it easier. If there is a way that can prune these models with reconstructing them i guess this will benefit the community in a greater way.
